src/julia-parser.scm

```diff
@@ -2285,10 +2285,14 @@
                       (if (closing-token? t)
                           (error (string "unexpected \"" (take-token s) "\"")))))
            (take-token s)
-           (if (and (eq? t 'var) (eqv? (peek-token s) #\") (not (ts:space? s)))
+           (if (and (eq? t 'var)
+                    (if (not (or (ts:pbtok s) (ts:last-tok s)))
+                      ;; Hack to preserve (io.pos (ts:port s)) for non-greedy Meta.parse
```

Comment:
Indentation is off here

Reply:
Oops thanks. I've adjusted my autoindent settings.